uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss

```diff
@@ -7,6 +7,14 @@
 
 // Customizations specific for T&A
 .civihr-ui-select {
+  * {
```

Comment:
is this really necessary? it's quite a wide net you are casting here

Reply:
The original style it is overriding is https://github.com/civicrm/civihr/blob/PCHR-3061-ta-admin-styles/org.civicrm.bootstrapcivihr/scss/components/civihr-ui-select/_overrides.scss#L57.

But I have changed to code to target the only selector which was not having proper border color.
```
  .ui-select-match-item {
    border-color: $brand-primary !important;
  }
```